docs/user/user_doc.md

```diff
@@ -0,0 +1,151 @@
+
+```python
+# Run Synchronously
+result = pipeline.run("This is a sample text")
+
+# Run Asynchronously
+## Start async job
+start_response = pipeline.run_async("This is a sample text")
+poll_url = start_response["url"]
+## Poll to see current job status
+poll_response = pipeline.poll(poll_url)
+```
+
+## Datasets
+aiXplain has an extensive collection of Datasets for various tasks like Translation, Speech Recognition, Diacritization, Sentiment Analysis and much more.
+You can even upload your own dataset on the platform [here](https://platform.aixplain.com/dashboard/datasets/upload).
+
+The catalog of all available datasets can be accessed and browsed on the aiXplain platform [here](https://platform.aixplain.com/discovery/datasets).
+
+The aiXtend SDK allows search for existing datasets that match a specific criteria. `DatasetFactory` can  search for datasets that are linked to a particular machine learning task and optionally support a specific input/output language pair. 
+
+```python
+from aixtend.factories.dataset_factory  import DatasetFactory
+dataset_list = DatasetFactory.get_first_k_assets(k=5, task="translation", input_language="en", output_language="fr")
+```
+Note: This does not download datasets to your local machine.
+
+## Metrics
+aiXplain has an impressive library of metrics for various machine learning tasks like Translation, Speech Recognition, Diacritization and Sentiment Analysis. We have reference similarity metrics, human evaluation estimation metrics, and referenceless metrics.
+
+The catalog of all available metrics can be accessed and browsed on the aiXplain platform [here](https://platform.aixplain.com/discovery/metrics).
+
+The aiXtend SDK allows search for existing metrics. `MetricFactory` can  search for metrics for a particular machine learning task.
+
+```python
+from aixtend.factories.metric_factory  import MetricFactory
+metric_list = MetricFactory.list_assets(task="translation")
+```
+
+## Benchmarks
+
+The [aiXplain platform](https://aixplain.com/platform/benchmark) provides a powerful set of tools for benchmarking machine learning models and evaluating their performance on specific tasks. You can obtain easy-to-interpret granular insights on the performance of models for quality, latency, footprint, cost, and bias with our interactive Benchmark reports.
+
+The proposed benchmarking framework is designed for being modular and interoperable in its core across three main components (models, datasets, metrics) in its body to become a one-stop shop for all possible benchmarking activities across several domains and metrics. You need to choose these three components for the task of your choice. 
+
+*Currently Supported tasks are*:  Translation, Speech Recognition, Diacritization and Sentiment Analysis with many more in works.
```

Comment:
nit: The currently supported functions are Translation, Speech Recognition, Diacritization and Sentiment Analysis. More supported functions will be added with further releases.

Why do we name the Benchmarking parameter as 'task' instead of 'function'?